build-constraints.yaml

```diff
@@ -5242,6 +5241,10 @@ packages:
         - curly-expander
         - lsql-csv
 
+    "Sebastian Graf <sgraf1337@gmail.com> @sgraf812":
+        - happy-lib
+        - happy < 1.21 # Note, cannot use 1.21.0
```

Comment:
Would be good to keep the stackage issue link in here for the stackage curators.